FlowCrypt/src/main/java/com/flowcrypt/email/api/retrofit/response/model/OrgRules.kt

```diff
@@ -193,16 +197,22 @@ data class OrgRules constructor(
       throw IllegalStateException("Not a valid email $emailAddr")
     }
 
-    val disallowedDomains = disallowAttesterSearchForDomains ?: emptyList()
-    return !disallowedDomains.any { it.equals(userDomain, true) }
+    val allowedDomains = allowAttesterSearchOnlyForDomains ?: emptyList()
```

Comment:
Careful, there is a huge difference between a null and empty array.

null means that option is not set - behave as before.

Empty array means it is set, and attester search is not allowed on any domain.

Should add a test that makes sure `allow_attester_search_only_for_domains: []` means don't allow on any.

Reply:
Good one. It would be nice to have this explanation in https://github.com/FlowCrypt/flowcrypt-ios/issues/1668 and https://flowcrypt.com/docs/business/org-rules.html